src/components/RichTextEditor/plugins/heading/HeadingElement.tsx

```diff
@@ -13,8 +13,9 @@ export const HeadingElement: ElementRenderer = (props) => {
   const { element, attributes, children } = props;
   if (element.type === "heading") {
     const El = `h${element.level}` as const;
+    const style = element.level === 2 ? "heading.medium" : "heading.small";
```

Comment:
Vi bruker kanskje ikke H1 men føles litt tryggere 😄 
```suggestion
    const style = element.level <= 2 ? "heading.medium" : "heading.small";
```